examples/webgl_postprocessing_unreal_bloom_selective.html

```diff
@@ -125,7 +125,7 @@
 
 			var mouse = new THREE.Vector2();
 
-			window.addEventListener( 'click', onDocumentMouseClick, false );
+			window.addEventListener( 'pointerdown', onDocumentPointerDown, false );
```

Comment:
Nit: How about renaming `onDocumentPointerDown()` do just `onPointerDown()`? `document` is not relevant here.

Reply:
yeah that makes more sense. I was trying to change as little as possible initially